src/prompt_systematic_review/load.py

```diff
@@ -0,0 +1,32 @@
+from datasets import load_dataset
+
+
+def load_hf_dataset(dataset_name, name=None, split=None):
+    """
+    Load in a Hugging Face dataset.
+
+    :param dataset_name: The name of the Hugging Face dataset to load.
+    :type dataset_name: str
+    :param name: Defines the name of the dataset configuration
+    :type name: str
+    :param split: Which split of the data to load. If None, will return a dict with all splits (typically datasets.Split.TRAIN and datasets.Split.TEST).
+    :type split: str or Split
+    :return: The loaded dataset.
+    :rtype: Dataset or DatasetDict
+    """
+
+    try:
+        if name:
```

Comment:
Double if is interesting :)

Reply:
I've gotten rusty : ). Made it all one line.